cloud_utils/scheduler/scheduler_test.py

```diff
@@ -33,3 +33,14 @@ def test_empty_scheduler():
         True,
         json.load((pathlib.Path(__file__).parent / "test_schedule_empty.json").open()),
     )
+
+
+# @pytest.mark.skip("ENG-2899")
```

Comment:
can we remove the skip ?

Reply:
Not yet.
It will fail because of cluster access. Let's prioritize the task to fix.